pkg/volume/glusterfs/glusterfs.go

```diff
@@ -68,6 +76,10 @@ const (
 	annGlusterUserKey           = "glusterfs.kubernetes.io/userkey"
 	annGlusterUser              = "glusterfs.kubernetes.io/userid"
 	gciGlusterMountBinariesPath = "/sbin/mount.glusterfs"
+	defaultGidMin               = "2000"
+	defaultGidMax               = "4294967295"
+	annDynamicallyProvisioned   = "pv.kubernetes.io/provisioned-by"
+	annStorageClass             = "volume.beta.kubernetes.io/storage-class"
```

Comment:
shouldn't we already know the storage class we're supposed to use? how were we getting our provisioning params before?